pkg/cli/evaluate.go

```diff
@@ -92,12 +98,33 @@ func updatePolicyEngineObjectsFromDirPath(pe *eval.PolicyEngine, podNames []type
 	for i := range objectsList {
 		obj := objectsList[i]
 		switch obj.Kind {
+		// workloads kinds
 		case parser.Pod:
 			err = pe.InsertObject(obj.Pod)
+		case parser.ReplicaSet:
+			err = pe.InsertObject(obj.ReplicaSet)
+		case parser.Deployment:
+			err = pe.InsertObject(obj.Deployment)
+		case parser.DaemonSet:
+			err = pe.InsertObject(obj.DaemonSet)
+		case parser.StatefulSet:
+			err = pe.InsertObject(obj.StatefulSet)
+		case parser.ReplicationController:
+			err = pe.InsertObject(obj.ReplicationController)
+		case parser.Job:
+			err = pe.InsertObject(obj.Job)
+		case parser.CronJob:
+			err = pe.InsertObject(obj.CronJob)
+			// ns kind
```

Comment:
if `eval` only supports pods, why adding these resources?
If the dir path only contains workload manifests and not pods, how would one infer the pod name for this command?
maybe this should be a separate issue, to extend eval support for workloads, but in this PR can just focus on pods?
the tests automation can add the pods resources and adjust the input, as if those manifests would exist for testing.

Reply:
not sure understood last sentence;
my tests dir paths contain manifests with StatefulSet workloads (I used replica name as the input of the test (pod name);
The tests failed without adding these cases (my manifests were not inserted to the policy-engine because of the lack of these cases; );
should I revert these changes and create new tests with pods only in the manifests?

Reply:
for these tests to work, you can add test automation (for eval tests) that add pods resources as if they existed in the manifests.. the changes here better be removed, until eval actually supports such resources, because in the current eval usage workload resources are not supported, and a user cannot infer the pod name to be used in this command when providing only workload resources.

Reply:
modified and added tests:
1. `command_test.go` : since `command-line` needs the "dir" as an argument to execute the cmd; 
for those tests (which don't contain pods in the manifests), I've generated a "temp dir" which is a copy of the original dir + generated yamls for pods (for the given src + dst arguments) so the command runs on the "temporary dir" successfully. 
_generating pod yaml for a workload considers also its labels (which may be used in the netpols rules and are essential for computing the result);_ 


2. `eval_test.go` uses the internal `eval` funcs, so it works successfully with the original dirs.
in this file we had tests from parsed resources (that tests the "eval" functionality) ;
 and I have added eval tests from dir paths.

Reply:
ok, please open a separate issue to consider supporting `eval` with input workloads, not just pods, and also add a comment for this test functionality added here, that it can be removed when this `eval` extension is added.

Reply:
done, new issue #440 ;  + comments were added in the relevant files